src/lib.js

```diff
@@ -1440,7 +1440,9 @@ module.exports = {
             for(const stat in pet.stats)
                 output.pet_bonus[stat] = (output.pet_bonus[stat] || 0) + pet.stats[stat];
         }
-
+        // for angler armors full set ability being wrong
+        if(items.armor.filter(a => getId(a).startsWith('Angler_Armor_')).length == 4)
+            
```

Comment:
Obviously this wont do anything by itself this is is just a start to solve hypickles idoticy